.github/workflows/buildtest.yml

```diff
@@ -60,7 +60,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        distro: ["debian-10", "debian-11", "ubuntu-18.04", "ubuntu-20.04"]
+        distro: ["debian-10", "debian-11", "ubuntu-20.04"]
```

Comment:
I would commit this separately :-)

Reply:
https://github.com/moves-rwth/storm/commit/6a26fb7115cfce8dd1ac3388987e6207b185bd9c